src/consent-manager.js

```diff
@@ -123,9 +128,10 @@ export default class ConsentManager {
     }
 
     saveConsents(){
-        if (this.consents === null)
+        var v = this.consents
+        if (v === null)
             deleteCookie(this.cookieName)
-        const v = JSON.stringify(this.consents)
+        v = v !== null ? JSON.stringify(this.consents) : ""
```

Comment:
This reassignment of `v` seems a bit too wild.
A variable should only have one type (and a speaking name, while you're at it).

Reply:
This fix is for older browsers: JSON.parse / JSON.stringify en empty strings results in a syntax error (tested on IE 9). But I see: `const` should be replaced with `var?

Reply:
fixed with: https://github.com/mkkeck/coi/commit/bf9bcd4cf5546bdebd8344c377c6ccb18fc7644b